codes/java/chapter_doublePointer/double_pointer.java

```diff
@@ -0,0 +1,44 @@
+

```

Comment:
Create a file header with the Date and Your ID. You can refer to other java files.